### DISCUSSION TITLE
Is it possible to ask Optuna to duplicate a gridsearch N times?

### DISCUSSION THREAD

**Discussion by HStromfelt** [optuna/optuna#4872] [Q&A] [answered]
Hey all,

I want to use Optuna when doing comprehensive studies over different hyperparameters, to not only find optimal values, but to assess their significance. 
I can see grid search is possible with Optuna. However, I need to perform N retries so I can see what kind of standard deviation I get in performance for each hyperparameter configuration.

Is this possible with Optuna?

**Comment by contramundum53** [comment, marked as answer]
We currently don't have a support for that yet. Some possible workarounds:
1. Manually `enqueue_trial` all possible combinations multiple times.
2. Create multiple studies, run grid search in each of them, and merge in the end.
3. In your objective function, run the evaluation multiple times.
4. Copy the `GridSampler` or `BruteForceSampler` and create your own sampler.
